rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/170_knn_search_hex_encoded_byte_vectors.yml

```diff
@@ -190,10 +190,16 @@ setup:
       index:
         index: knn_hex_vector_index_dyn_dims
         id: "1"
+        refresh: true
         body:
           my_vector_byte: "807f0a"
           my_vector_byte_indexed: "807f0a"
 
+  # wait and ensure that the index update is replicated and searchable
```

Comment:
> wait and ensure that the index update is replicated and searchable

Does the `refresh` above not guarantee this? In case this is related to replication, should we set the number of replicas to 0 instead?

Reply:
@carlosdelest it doesn't, I just remember us running into this in the past, and we wait for a cluster update to occur and that fixed it.

Reply:
So this is related to mapping updates alone, right? `refresh` alone should suffice for making docs available for search after indexing?

Reply:
@carlosdelest correct. it has to do with making sure the update is visible in the following get request.